package-lock.json

```diff
@@ -19023,6 +19023,11 @@
         }
       }
     },
+    "utility-types": {
+      "version": "3.10.0",
+      "resolved": "https://registry.npmjs.org/utility-types/-/utility-types-3.10.0.tgz",
+      "integrity": "sha512-O11mqxmi7wMKCo6HKFt5AhO4BwY3VV68YU07tgxfz8zJTIxr4BpsezN49Ffwy9j3ZpwwJp4fkRwjRzq3uWE6Rg=="
+    },
```

Comment:
This gets us the `$Diff` flow type. The flot-to-ts conversion script relies on `utility-types` to support some existing flow code without having to completely rewrite it.